etna/models/holt_winters.py

```diff
@@ -609,18 +616,47 @@ def __init__(
             )
         )
 
+    def params_to_tune(self) -> Dict[str, "BaseDistribution"]:
+        """Get default grid for tuning hyperparameters.
+
+        This grid doesn't tune ``seasonal_periods`` parameter. It expected to be set by the user.
 
-class HoltModel(HoltWintersModel):
+        Returns
+        -------
+        :
+            Grid to tune.
+        """
+        if self.seasonal is None:
+            return {
+                "trend": CategoricalDistribution(["add", "mul", None]),
+                "damped_trend": CategoricalDistribution([False, True]),
+                "use_boxcox": CategoricalDistribution([False, True]),
+            }
+        else:
+            return {
+                "trend": CategoricalDistribution(["add", "mul", None]),
+                "damped_trend": CategoricalDistribution([False, True]),
+                "seasonal": CategoricalDistribution(["add", "mul", None]),
+                "use_boxcox": CategoricalDistribution([False, True]),
+            }
+
+
+class HoltModel(
+    PerSegmentModelMixin,
```

Comment:
If we keep inherit `HoldModel` from `HoltWintersModel` we will get the error during `to_dict` because we don't save parameter `exponential` anywhere inside `HoldModel`. It is concerning that we haven't face this error before.